flac.md

```diff
@@ -814,9 +856,46 @@ The following information serves as the registration form for the "audio/flac" m
 
 FLAC owes much to the many people who have advanced the audio compression field so freely. For instance:
 
-- [A. J. Robinson](https://web.archive.org/web/20160315141134/http://mi.eng.cam.ac.uk/~ajr/) for his work on Shorten; his paper ([@robinson-tr156]) is a good starting point on some of the basic methods used by FLAC. FLAC trivially extends and improves the fixed predictors, LPC coefficient quantization, and Rice coding used in Shorten.
-- [S. W. Golomb](https://web.archive.org/web/20040215005354/http://csi.usc.edu/faculty/golomb.html) and Robert F. Rice; their universal codes are used by FLAC's entropy coder.
-- N. Levinson and J. Durbin; the reference encoder uses an algorithm developed and refined by them for determining the LPC coefficients from the autocorrelation coefficients.
-- And of course, [Claude Shannon](https://en.wikipedia.org/wiki/Claude_Shannon)
+- A. J. Robinson for his work on Shorten; his paper (see [@robinson-tr156]) is a good starting point on some of the basic methods used by FLAC. FLAC trivially extends and improves the fixed predictors, LPC coefficient quantization, and Rice coding used in Shorten.
+- S. W. Golomb and Robert F. Rice; their universal codes are used by FLAC's entropy coder, see [@Rice].
+- N. Levinson and J. Durbin; the FLAC reference encoder (see [implementation status](#implementation-status)) uses an algorithm developed and refined by them for determining the LPC coefficients from the autocorrelation coefficients, see [@Durbin].
+- And of course, Claude Shannon, see [@Shannon].
+
+<reference anchor="Rice" target="https://ieeexplore.ieee.org/document/1090789">
+    <front>
+        <title>Adaptive Variable-Length Coding for Efficient Compression of Spacecraft Television Data</title>
+        <author initials="RF" surname="Rice" fullname="Robert Rice">
+            <organisation>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA, USA</organisation>
+        </author>
+        <author initials="JR" surname="Plaunt">
+            <organisation>Jet Propulsion Laboratory, California Institute of Technology, Pasadena, CA, USA</organisation>
```

Comment:
use US spelling: organisation > organization

Reply:
Strange that xml2rfc didn't complain about this

Reply:
(I personally prefer the UK spelling. But it should be consistent either way.)